docs/source/user-guide/sql/explain.md

```diff
@@ -21,41 +21,219 @@
 
 The `EXPLAIN` command shows the logical and physical execution plan for the specified SQL statement.
 
-See the [Reading Explain Plans](../explain-usage.md) page for more information on how to interpret these plans.
+## Syntax
 
 <pre>
-EXPLAIN [ANALYZE] [VERBOSE] statement
+EXPLAIN [ANALYZE] [VERBOSE] [FORMAT format] statement
```

Comment:
Maybe also state here that using `VERBOSE` and `FORMAT` together is not supported (yet?) like `ANALYZE` and `FORMAT` below

Reply:
I will do so

Reply:
I tried to clarify in 398cf1824